notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1994 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Create the repository in Artifact registry
</pre><p>You can probably skip creating a repository and just move to the next step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
Seems like the repository creation step is necessary. Otherwise, Cloud Build submit throws an error saying the repository doesn't exist. Following the docs from [here](https://cloud.google.com/build/docs/build-push-docker-image).